### ISSUE TITLE
[TRIGGER] ClickUp sources

### ISSUE THREAD

**Issue by sergioeliot2039** [PipedreamHQ/pipedream#3972]
**App**
ClickUp

**Sources**
New Task
Updated Task
New Folder
New List

Webhooks available, see API docs at https://jsapi.apiary.io/apis/clickup20/introduction/faq.html
Updated Task requested by user at https://github.com/PipedreamHQ/pipedream/issues/3950

**Comment by dylburger** [comment]
Thanks for the request! Did you know that anyone can [author and publish their own Pipedream sources](https://pipedream.com/docs/components/quickstart/nodejs/sources/)? Would you be interested in helping out with that? We're happy to answer any questions or support you in any way.

If you don't have the time, would you be willing to reach out to the app provider to see if they're interested in building these? When we work with an app partner, it improves the quality of the integration, and it helps when this kind of request comes from a customer like you.

**Comment by ljdatasci** [comment]
I can reach out to the app provider and ask if they want to build these components.

**Comment by sergioeliot2039** [comment]
hey @ljdatasci, thanks for your availability!

@ctrlaltdylan or @todsac do you know how we can on board @ljdatasci to build components? (unsure if there's an screening process for users to help)

he seems keen to develop ClickUp components.

**Comment by ljdatasci** [comment]
I commited a new Action for the Notion API this past week. I read the PipeDream Components API documentation also.

**Comment by ljdatasci** [comment]
I'm ready, and I'm a she, Thanks.

**Comment by ctrlaltdylan** [comment]
I'm sorry @ljdatasci, didn't know that. Noted!

Thanks for waiting on me, I've double your included daily invocations, on the house of course no charge.

Excited to see what you build! And we're always happy to hear your feedback on the component development experience, as well as if there are any gaps in docs that we need to help answer.

**Comment by ljdatasci** [comment]
@ctrlaltdylan. Can you please assign this issue to me under Assignees? Thanks.

**Comment by ljdatasci** [comment]
@sergioeliot2039, @ctrlaltdylan  In order create any of the ClickUp sources, the user needs access to webhooks.  Webhooks can then be used to send the status of ClickUp events such as New Task, New List, New Folder, etc.  Webhooks require a ClickUp Teams account, which I don't have access to, and I do all of my PRs through my personal account. So, I'm going to pass on this PR.

**Comment by vellames** [comment]
@ljdatasci Hi there!
Are you still working on this? I assigned it for me but realized you are already working on it. If so, please assign it to you :)

**Comment by ljdatasci** [comment]
@vellames I need a business account in order to test the actions, and I do all of my testing through my personal account.  So you can work on this PR. Thanks.
